media/videostitcher/stitcher_test.go

```diff
@@ -186,23 +200,82 @@ func TestCdnKeys(t *testing.T) {
 	}
 
 	// Tests
-	// Google CDN tests
+	// Media CDN tests
+
+	// Create a new Media CDN key.
+	testutil.Retry(t, 3, 2*time.Second, func(r *testutil.R) {
```

Comment:
Do we know that this test will be flaky? If not, then I don't think we should put it inside of a call to `testutil.Retry()`.

Reply:
Calls to the API have been somewhat flaky in the near past, so I'd like to keep as is.